static/js/partners.js

```diff
@@ -1,35 +1,44 @@
-/**
- * Generate a warnin only if warnings are available.
- */
-function warn(message) {
```

Comment:
We still need to worry about the same set of browsers don't we? To me moving to ES5 just means adding a bunch of polyfills. This could easily be one of them.